episode-0162/README.md

```diff
@@ -0,0 +1,27 @@
+# Episode 162: January 16th, 2019
+
+## Links
+* [Watch this episode on new Air Mozilla](https://air.mozilla.org/event-redirect/318580/)
+* [Evernote Agenda](https://www.evernote.com/shard/s434/client/snv?noteGuid=534eeccf-3988-4e1c-ac4b-3acde3fb991a&noteKey=ad00fb501fe48556&sn=https%3A%2F%2Fwww.evernote.com%2Fshard%2Fs434%2Fsh%2F534eeccf-3988-4e1c-ac4b-3acde3fb991a%2Fad00fb501fe48556&title=January%2B16th%252C%2B2019%2B-%2BEpisode%2B162)
+
+## Topics
+  - 
+Happy New Year!
```

Comment:
Thanks for doing this! For the next few, I think we can drop the Happy New Year. :)